tests/web/APIRest.php

```diff
@@ -76,8 +77,12 @@ protected function query(
       $params = [],
       $expected_code = 200,
       $expected_symbol = '',
-      bool $file = false
+      bool $no_decode = false
    ) {
+      if (!is_array($expected_code)) {
+         $expected_code = [$expected_code];
+      }
+
```

Comment:
Changed expected_code to an array as some call may return different valid codes like the search:
- 200
- 206 for partial content

Since the items created during the web tests are not removed from the database after their execution, we may not know if have enough items to get a 206.